project/Version.scala

```diff
@@ -15,7 +15,7 @@
  */
 
 object Version {
-  val geotrellis  = "3.0.1-LOCAL"
+  val geotrellis  = s"3.0.1${Environment.versionSuffix}"
```

Comment:
I think `3.0.1` is local testing leaking. We haven't published `3.0.0` yet so I think we should be fine to have this as `3.0.0-SNAPSHOT` ?

Reply:
@echeipesh  a good point